app/api/__main__.py

```diff
@@ -21,14 +23,15 @@ def main() -> None:
     logger.info("Running API Application")
 
     app = api.app.create_app()
+    port = os.getenv("PORT", 8080)
```

Comment:
Could we put this env var in the app config object?

(I meant to do this when moving it over, but looks like I forgot)

Reply:
Done!